README.md

```diff
@@ -5,4 +5,6 @@ A KFService encapsulates the complexity of autoscaling, networking, health check
 
 In the future, we hope to support more advanced use cases such as outlier detection, bias detection, explainability, pre/post processing, and performance profiling across infrastructure configurations.
 
-This project is an evolution of the [original proposal in the Kubeflow repo](https://github.com/kubeflow/kubeflow/issues/2306). To know more about KFServing, please [read the docs](/docs)
\ No newline at end of file
+This project is an evolution of the [original proposal in the Kubeflow repo](https://github.com/kubeflow/kubeflow/issues/2306). To know more about KFServing, please [read the docs](/docs)
+
+To deploy, develop and test KFServing on top of your Knative deployment, please [follow this doc](/docs/DEVELOPER_GUIDE.md).
```

Comment:
Can we drop "on top of Knative deployment"?

"For developers looking to contribute, please [follow this doc]"

Reply:
Done